doc/source/example-streaming.rst

```diff
@@ -0,0 +1,73 @@
+Streaming MapReduce
+===================
+
+This document walks through how to implement a simple streaming application
+using Ray's actor capabilities. It implements a streaming MapReduce which
+computes word counts on wikipedia articles.
```

Comment:
Would be good to maybe give some example text and write what the desired output should be (and maybe the desired intermediate computations) so that people know what is going on here